dipy/direction/closest_peak_direction_getter.pyx

```diff
@@ -56,7 +56,7 @@ cdef int closest_peak(np.ndarray[np.float_t, ndim=2] peak_dirs,
             return 0
     return 1
 
-cdef class BaseDirectionGetter(DirectionGetter):
+cdef class BasePmfDirectionGetter(DirectionGetter):
```

Comment:
`BaseDirectionGetter` needs a deprecation cycle as you did for `EuDXDirectionGetter`. 

Can you add this deprecation and a small test for it? 

Thank you